### PR TITLE
WIP: Add private key URI standard

### PR DESCRIPTION
**Pull request by ligi**
WIP as this needs a way to also contain json/UTC encoded keys - but not yet sure about the encoding - base64 or url encoded? anyway it is getting kind of huge and QR codes a bit hard to scan this way

### CODE REVIEW

File: EIPS/keys-uri.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+    eip:
```
Comment (Arachnid) [inline EIPS/keys-uri.md]: Please number this 1001 and rename to eip-1001.md.

File: EIPS/keys-uri.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+---
```
Comment (Arachnid) [inline EIPS/keys-uri.md]: Elements in the header should not be indented.

File: EIPS/keys-uri.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    eip: 
+    title: URL Format for private keys
+    status: Active
+    type: Standards Track
+    author: @ligi
```
Comment (Arachnid) [inline EIPS/keys-uri.md]: Please include a handle - eg, `Ligi Ligison (@ligi)`.

File: EIPS/keys-uri.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
+---
+    eip: 
+    title: URL Format for private keys
+    status: Active
```
Comment (Arachnid) [inline EIPS/keys-uri.md]: This should be 'Draft'.

**Comment by jagnarong** [comment]
Good

**Comment by ligi** [comment]
Yes I will provide examples - plan to finish this EIP soon and examples will be added then.

File: EIPS/eip-1001.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
+---
+eip: 1001
+title: URL Format for private keys
+status: Draft
+type: Standards Track
```
Comment (nicksavers) [inline EIPS/eip-1001.md]: This is missing **Category: ERC**

File: EIPS/eip-1001.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+author: ligi (@ligi)
+created: 2018-04-015
```
Comment (nicksavers) [inline EIPS/eip-1001.md]: The day in a date should consist of 2 digits.

**Comment by axic** [comment]
@ligi are you still pursuing this?

**Comment by axic** [comment]
Ping @ligi

**Pull request closed without merging**